test/test_external.py

```diff
@@ -173,7 +173,7 @@ def test_english_to_spanish(self):
     def test_sentiment_model(self):
         io = gr.Interface.load("models/distilbert-base-uncased-finetuned-sst-2-english")
         try:
-            output = io("I am happy, I love you.")
+            output = io("I am happy, I love you")
```

Comment:
I think we're still hitting the cache in the api server that's returning the "old" List[Dict] prediction as opposed to the correct List[List[Dict]] prediction.